docs/teams/teams_fllc.md

```diff
@@ -1,21 +1,21 @@
 # Swerve First LEGO League Challenge Teams
 
-(FLLC logo here)
```

Comment:
Will the width adjust the size? It currently shows up in the preview.

Reply:
I updated it to be 15%.